src/main/java/core/basesyntax/GeneratorFigure.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeneratorFigure {
+    private static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```

Comment:
```suggestion
    private static final int COUNT_OF_FIGURE = 5;
```
we can implement only inteface, Figure is a class